sdktests/common_tests_events_contexts.go

```diff
@@ -51,7 +51,6 @@ func makeEventContextTestParams() []eventContextTestParams {
 					b.Kind("org")
 					b.Name("a")
 					b.SetString("b", "c")
-					b.Anonymous(true)
```

Comment:
Why this change?

Reply:
These tests are used for PHP and non-PHP server side SDKs.

If the SDK is PHP, it takes a branch that tries to validate the context provided in that event. You can change what that expectation looks like but it also affects the index event.

Non-PHP doesn't have this same branch. By removing the anonymous flag, the feature event context won't be different from the index one, and I don't have to change a ton of stuff.

Also, I have a test verifying the anonymous behavior of PHP with feature events, so no coverage is lost.